Backend/Backend/Services/Interfaces/IPodcastService.cs

```diff
@@ -32,6 +32,6 @@ public interface IPodcastService
     public Task<bool> EditEpisodeTranscriptLinesAsync(User user,Guid episodeId, TranscriptLineResponse[] lines);
     public Task<ListenPositionResponse> GetWatchHistory(User user, Guid episodeId);
     public Task<AdjecentEpisodeResponse> GetAdjecentEpisodeAsync(Guid episodeId);
-    public Task<object?> GetMetrics(User user, Guid podcastId);
+    public Task<object?> GetMetrics(User user, Guid podcastId, string domainUrl);
```

Comment:
Creating a custom Response definition might be better than having type object?. It would definitely make the code more robust.

Reply:
Done!